tools/topology/sof/tokens.m4

```diff
@@ -117,3 +117,11 @@ SectionVendorTokens."sof_led_tokens" {
 	SOF_TKN_MUTE_LED_USE			"1300"
 	SOF_TKN_MUTE_LED_DIRECTION		"1301"
 }
+
+SectionVendorTokens."sof_chmap_tokens" {
+	SOF_TKN_STREAM_MAP_NUM_CH_MAPS		"1400"
```

Comment:
this has to be split into a separate token maybe called sof_mux_demux_token

Reply:
I mean sure... So what is the reasoning for this? It could be that the channel maps etc are used elsewhere also than demux only. I mean maybe it would be somehow more clear, but basically we are defining the "parse unit" in the kernel side tuples array definitions, so that could easily combine tuple id's all over and no one would notice...